sky/packages/sky_engine/BUILD.gn

```diff
@@ -137,6 +137,11 @@ copy("math") {
       [ "$root_gen_dir/dart-pkg/sky_engine/lib/math/{{source_file_part}}" ]
 }
 
+copy("spirv") {
+  sources = [ "//flutter/lib/spirv/lib" ]
```

Comment:
I think it works both ways because the output is just a single library (spirv.dart), but changed to match others for more consistency.

Reply:
This one is different because the file that it's copying is a directory - I didn't see guidance against that, but figured it would work well here since this folder is already a valid Dart library on its own (no need to selectively pick out specific files) For that reason there's no need to maintain a list of the files/subdirectories in the folder, but we can change to that if you recommend it.